lib/wallet/walletdb.js

```diff
@@ -482,6 +482,27 @@ class WalletDB extends EventEmitter {
     return this.scan(height);
   }
 
+  /**
+   * Get MTP for a block hash.
+   * @param {Hash} blockhash
+   * @returns {Promise}
+   */
+
+  async getMedianTime(blockhash) {
+    let time = null;
+
+    const raw = await this.db.get(layout.m.encode(blockhash));
+
+    if (raw) {
+      time = raw.readUInt32LE(0, true);
+    } else {
+      time = await this.client.getMedianTime(blockhash);
```

Comment:
I'm getting an error here:
`[error] (node) TypeError: this.client.getMedianTime is not a function`